commons/src/test/java/org/eclipse/kapua/commons/event/ServiceEventClientConfigurationTest.java

```diff
@@ -0,0 +1,91 @@
+        assertEquals("not_equals", serviceEventBusListener, serviceEventClientConfiguration.getEventListener());
+    }
+
+    @Test
+    public void constructorClientNameNullTest() {
+        ServiceEventClientConfiguration serviceEventClientConfiguration = new ServiceEventClientConfiguration("address", null, serviceEventBusListener);
+        assertEquals("not_equals", "address", serviceEventClientConfiguration.getAddress());
+        assertNull("not_null", serviceEventClientConfiguration.getClientName());
+        assertEquals("not_equals", serviceEventBusListener, serviceEventClientConfiguration.getEventListener());
+    }
+
+    @Test
+    public void constructorServiceEventNullTest() {
+        ServiceEventClientConfiguration serviceEventClientConfiguration = new ServiceEventClientConfiguration("address", "subscriberName", null);
+        assertEquals("not_equals", "address", serviceEventClientConfiguration.getAddress());
+        assertEquals("not_equals", "subscriberName", serviceEventClientConfiguration.getClientName());
+        assertNull("not_null", serviceEventClientConfiguration.getEventListener());
+    }
+
+    @Test
+    public void constructorAddressAndNameNullTest() {
+        ServiceEventClientConfiguration serviceEventClientConfiguration = new ServiceEventClientConfiguration(null, null, serviceEventBusListener);
+        assertNull("not_null", serviceEventClientConfiguration.getAddress());
+        assertNull("not_null", serviceEventClientConfiguration.getClientName());
+        assertEquals("not_equals", serviceEventBusListener, serviceEventClientConfiguration.getEventListener());
+    }
+
+    @Test
+    public void constructorNameAndServiceEventNullTest() {
+        ServiceEventClientConfiguration serviceEventClientConfiguration = new ServiceEventClientConfiguration("address", null, null);
+        assertEquals("not_equals", "address", serviceEventClientConfiguration.getAddress());
+        assertNull("not_null", serviceEventClientConfiguration.getClientName());
+        assertNull("not_null", serviceEventClientConfiguration.getEventListener());
+    }
+
+    @Test
+    public void constructorAddressAndNameCharCheckTest() {
+        String[] permittedValues = {"", "!@#$%^^&**(-()_)+/|", "regularNaming", "regular Naming", "49", "regularNaming49", "NAMING", "246465494135646120009090049684646496468456468496846464968496844"};
+        for (String value : permittedValues) {
+                ServiceEventClientConfiguration serviceEventClientConfiguration = new ServiceEventClientConfiguration(value, value, serviceEventBusListener);
+                assertEquals("not_equals", value, serviceEventClientConfiguration.getAddress());
+                assertEquals("not_equals", value, serviceEventClientConfiguration.getClientName());
+                assertEquals("not_equals", serviceEventBusListener, serviceEventClientConfiguration.getEventListener());
```

Comment:
Please change the message to: "Expected and actual values are not the same" or something similar. "not_equals" is not a valid comment.